modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/model/SupportedLanguage.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.translation.translator.deepl.internal.model;
+
+/**
+ * @author Yasuyuki Takeo
+ */
+public class SupportedLanguage {
+
+	public String getLanguage() {
+		return language;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public Boolean getSupports_formality() {
+		return supports_formality;
+	}
+
+	public void setLanguage(String language) {
+		this.language = language;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public void setSupports_formality(Boolean supports_formality) {
+		this.supports_formality = supports_formality;
+	}
+
+	public String language;
+	public String name;
+	public Boolean supports_formality;
```

Comment:
You need to make this fields private. Rename them, adding a '_' as a prefix (e.g. "_language"). Also, names need to be camel cased (e.g. "_supportsFormality").

Reply:
@adolfopa supports_formality is the definition of DeepL API. In that case, what am I supposed to do? Should I change the name to Liferay convention and map the DeepL API parameter name manually?

Reply:
Yes, our code conventions take precedence over that. You'll need to map the correct name.

Reply:
@adolfopa understood.